.env.meteo

```
@@ -0,0 +1 @@
+CONFIG_NAME=meteo-france
```

```suggestion
CONFIG_NAME=meteo
```
Ici + dir `./configs/meteo`. "meteo" étant déjà utilisé dans les scripts et dans `./src/custom`.
(moot si on passe par envvar)